app/js/lib/foxbox/services/ip-camera.js

```diff
@@ -14,10 +14,6 @@ export default class IpCameraService extends BaseService {
     return TYPE;
   }
 
-  getImageList() {
-    return this.get('image_list');
-  }
```

Comment:
+ it's a good first bug for any contributor that would like to work on the app :)